### PR TITLE
spider: update help

### PR DESCRIPTION
**Pull request by karthikuj**
Corresponding PR - https://github.com/zaproxy/zaproxy/pull/7356
Signed-off-by: Karthik UJ <karthikuj2001@gmail.com>

### CODE REVIEW

File: addOns/spider/src/main/javahelp/org/zaproxy/addon/spider/resources/help/contents/spider.html [hunk 1]
```diff
@@ -44,6 +44,8 @@ <h4>HTML</h4>
 			specified in the <a href="options.html">Options
 				Spider screen</a>
 		</li>
+		<li>Import - 'implementation' attribute</li>
+		<li>Inline string - 'p', 'title', 'li', 'h1', 'h2', 'h3', 'h4', 'h5', 'h6' and 'blockquote' tags</li>
```
Comment (kingthorin) [inline addOns/spider/src/main/javahelp/org/zaproxy/addon/spider/resources/help/contents/spider.html:48]: `h6`, and

add a comma please

Comment (karthikuj) [reply addOns/spider/src/main/javahelp/org/zaproxy/addon/spider/resources/help/contents/spider.html]: Done.

**Review by kingthorin — APPROVED** [review]

**Comment by thc202** [comment]
Changed to WIP pending the merge of core PR.

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Pull request merged**